public.tf

```diff
@@ -18,6 +18,7 @@ resource "aws_subnet" "public" {
     "Name"      = "${module.public_label.id}${var.delimiter}${element(var.names, count.index)}"
```

Comment:
Should we merge all of these tags with [label](https://github.com/cloudposse/terraform-aws-named-subnets/pull/4/files#diff-fb78b6708cfb596c9fced4d8492d07adR1) `tags`?